<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QSplitter::handle:vertical {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 0), 
stop:0.407273 rgba(200, 200, 200, 255), 
stop:0.4825 rgba(101, 104, 113, 235), 
stop:0.6 rgba(255, 255, 255, 0));
/*border: 1px solid #777;
width: 13px;
margin-top: 2px;
margin-bottom: 2px;
border-radius: 4px;*/
}

QSplitter::handle:horizontal {
	background-color: qlineargradient(spread:reflect, x1:0.514, y1:0.5, x2:0.50838, y2:1, stop:0 rgba(95, 95, 95, 255), stop:1 rgba(255, 255, 255, 0));

/*margin-top: 2px;
margin-bottom: 2px;
border-radius: 4px;*/
}

</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>545</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FFFFFF, stop: 1 #E0E0E0);
border-top: 1px solid  #E0E0E0;
}</string>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Yardım</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuAyarlar">
    <property name="title">
     <string>Bağlantı</string>
    </property>
    <addaction name="actionSerial_RTU"/>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuAyarlar"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777205</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QStatusBar{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E0E0E0, stop: 1 #FFFFFF);
border: 1px solid  lightgrey;
border-radius: 2px;
margin-top: 2px;
}
</string>
   </property>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/exit-16.png</normaloff>:/icons/exit-16.png</iconset>
   </property>
   <property name="text">
    <string>Çıkış</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave_Session">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/document-export-16.png</normaloff>:/icons/document-export-16.png</iconset>
   </property>
   <property name="text">
    <string>Dışa Aktar</string>
   </property>
  </action>
  <action name="actionLoad_Session">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/document-import-16.png</normaloff>:/icons/document-import-16.png</iconset>
   </property>
   <property name="text">
    <string>İçe Aktar</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/info-sign-16.png</normaloff>:/icons/info-sign-16.png</iconset>
   </property>
   <property name="text">
    <string>Hakkında</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSerial_RTU">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/serial-pot-16.png</normaloff>:/icons/serial-pot-16.png</iconset>
   </property>
   <property name="text">
    <string>Port Ayar</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionScan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/eye-scan32.png</normaloff>:/icons/eye-scan32.png</iconset>
   </property>
   <property name="text">
    <string>Tara</string>
   </property>
  </action>
  <action name="actionPersistentEventLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/logs.png</normaloff>:/icons/logs.png</iconset>
   </property>
   <property name="text">
    <string>Olay Kaydı</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/plug-disconnect-16.png</normaloff>
     <normalon>:/icons/plug-connect-16.png</normalon>:/icons/plug-disconnect-16.png</iconset>
   </property>
   <property name="text">
    <string>Bağlan</string>
   </property>
  </action>
  <action name="actionNew_RegAddress">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/1266458.png</normaloff>:/icons/1266458.png</iconset>
   </property>
   <property name="text">
    <string>Register Giriş</string>
   </property>
  </action>
  <action name="actionNew_Parameter">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/1266455.png</normaloff>:/icons/1266455.png</iconset>
   </property>
   <property name="text">
    <string>Register Ayarları(F6/F10)</string>
   </property>
   <property name="toolTip">
    <string>Writable Register Config</string>
   </property>
  </action>
  <action name="actionNew_ModbusFunction">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/1266432.png</normaloff>:/icons/1266432.png</iconset>
   </property>
   <property name="text">
    <string>Modbus Functions</string>
   </property>
   <property name="toolTip">
    <string>Modbus Functions</string>
   </property>
  </action>
  <action name="actionNewDeviceType">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/connect-24.png</normaloff>:/icons/connect-24.png</iconset>
   </property>
   <property name="text">
    <string>Cihaz Ekle/Kaldır</string>
   </property>
   <property name="toolTip">
    <string>Cihaz Ekle/Kaldır</string>
   </property>
  </action>
  <action name="actionNew_Page">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/1266432.png</normaloff>:/icons/1266432.png</iconset>
   </property>
   <property name="text">
    <string>Sayfa Ekle</string>
   </property>
  </action>
  <action name="actionNew_Menu">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/1266432.png</normaloff>:/icons/1266432.png</iconset>
   </property>
   <property name="text">
    <string>Menu Ekle</string>
   </property>
  </action>
  <action name="actionPair_MenuPage">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/1266457.png</normaloff>:/icons/1266457.png</iconset>
   </property>
   <property name="text">
    <string>Pair Menu - Page</string>
   </property>
  </action>
  <action name="actionWriteToDevice">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/upload.png</normaloff>:/icons/upload.png</iconset>
   </property>
   <property name="text">
    <string>Cihaza Yaz</string>
   </property>
   <property name="toolTip">
    <string>Cihaza Yaz</string>
   </property>
  </action>
  <action name="actionReadFromDevice">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/download.png</normaloff>:/icons/download.png</iconset>
   </property>
   <property name="text">
    <string>Cihazdan Oku</string>
   </property>
   <property name="toolTip">
    <string>Cihazdan Oku</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/onboarding.png</normaloff>:/icons/onboarding.png</iconset>
   </property>
   <property name="text">
    <string>Dosya Aç</string>
   </property>
   <property name="toolTip">
    <string>Dosya Aç</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Farklı Kaydet</string>
   </property>
   <property name="toolTip">
    <string>Farklı Kaydet</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Kaydet</string>
   </property>
   <property name="toolTip">
    <string>Kaydet</string>
   </property>
  </action>
  <action name="actionPair_RegisterPage">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/1266457.png</normaloff>:/icons/1266457.png</iconset>
   </property>
   <property name="text">
    <string>Pair Page - Register</string>
   </property>
   <property name="toolTip">
    <string>Pair Page With Registers </string>
   </property>
  </action>
  <action name="actionPair_TypeRegister">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/1266457.png</normaloff>:/icons/1266457.png</iconset>
   </property>
   <property name="text">
    <string>Pair  Type - Register</string>
   </property>
   <property name="toolTip">
    <string>Pair Type And Register</string>
   </property>
  </action>
  <action name="actionFactoryCalibration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/multimeter.png</normaloff>:/icons/multimeter.png</iconset>
   </property>
   <property name="text">
    <string>Kalibrasyon</string>
   </property>
   <property name="toolTip">
    <string>Fabrika Kalibrasyonu</string>
   </property>
  </action>
  <action name="actionAlarms">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/warning.png</normaloff>:/icons/warning.png</iconset>
   </property>
   <property name="text">
    <string>Alarm</string>
   </property>
   <property name="toolTip">
    <string>Alarm</string>
   </property>
  </action>
  <action name="actionDefineAlarmBits">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/1266455.png</normaloff>:/icons/1266455.png</iconset>
   </property>
   <property name="text">
    <string>Alarm Tanımları</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
